securedrop_client/gui/widgets.py

```diff
@@ -1277,8 +1279,8 @@ class FileWidget(QWidget):
     Represents a file.
     """
 
-    def __init__(self, source_db_object, submission_db_object,
-                 controller, file_ready_signal, align="left"):
+    def __init__(self, source_db_object: Source, submission_db_object: Any,
+                 controller: Controller, file_ready_signal: Any, align: str = "left") -> None:
```

Comment:
`submission_db_object`: `Any` -> `File`

Signal: `Any` -> `Function[...]` (you'll have to double check the args here).